Statistics/Statistics.cs

```diff
@@ -1,12 +1,28 @@
-﻿using System;
+﻿  
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Statistics
 {
+    
     public class StatsComputer
     {
-        public Stats CalculateStatistics(List<float> numbers) {
-            //Implement statistics here
+        public Stats CalculateStatistics(List<double> numbers)
+        {
+            Stats obj = new Stats();
+            if (numbers.Count == 0) {
+                obj.Max = double.NaN;
+                obj.Min = double.NaN;
+                obj.Average = double.NaN;
```

Comment:
Could you initialize these values in the Stats construction itself? Then you don't need to worry about uninitialized values anywhere in the code.